lib/rules/valid-slot-scope.js

```diff
@@ -66,7 +66,7 @@ module.exports = {
     docs: {
       description: 'enforce valid `slot-scope` attributes',
       category: undefined,
-      url: 'https://github.com/vuejs/eslint-plugin-vue/blob/v5.0.0-beta.4/docs/rules/valid-slot-scope.md'
+      url: 'https://github.com/vuejs/eslint-plugin-vue/blob/v5.0.0-beta.5/docs/rules/valid-slot-scope.md'
```

Comment:
@michalsnik what do you think about moving it to config than we don't have to modify it in during every release :)

instead
`'https://github.com/vuejs/eslint-plugin-vue/blob/v5.0.0-beta.5/docs/rules/valid-slot-scope.md'`
we will have
`${config.ruleUrl}/valid-slot-scope.md'
or something like that